rundeckapp/grails-app/domain/rundeck/ScheduledExecution.groovy

```diff
@@ -1154,13 +1154,13 @@ class ScheduledExecution extends ExecutionContext implements JobData, EmbeddedJs
         if(this.id) {
             def queryArgs = lock ? [lock: true]  : [:]
 
-            stats = ScheduledExecutionStats.findBySe(this, queryArgs)
+            stats = ScheduledExecutionStats.findByJobUuid(this.uuid)
```

Comment:
This direct access of ScheduledExecutionStats needs to be replaced with a provider access.
This method should probably be removed from this class and put in a service class instead.